cplusplus/UnconvertFromZDW.cpp

```diff
@@ -129,7 +132,7 @@ void InitDirAndBasenameFromFileName(string const& inFileNameStr, char * &sourceD
 namespace adobe {
 namespace zdw {
 
-const int UnconvertFromZDW_Base::UNCONVERT_ZDW_VERSION = 10;
+const int UnconvertFromZDW_Base::UNCONVERT_ZDW_VERSION = 11;
 const char UnconvertFromZDW_Base::UNCONVERT_ZDW_VERSION_TAIL[3] = "";
 
 const size_t UnconvertFromZDW_Base::DEFAULT_LINE_LENGTH = 16*1024; //16K default
```

Comment:
looks like I missed one in my previous whitespace cleanup!

Reply:
Missed what?  Care to clarify?

Reply:
Ah; whitespace around the multiply.  Fixed.